frontend/src/components/IndexPage.vue

```diff
@@ -9,7 +11,9 @@ import IconButton from './IconButton.vue';
 import { ButtonSizeEnum } from './IconButton.vue';
 import IconStarFilledVue from './icons/IconStarFilled.vue';
 import IconStarOutlinedVue from './icons/IconStarOutlined.vue';
-import { ref } from 'vue';
+
+import { ESize } from './enums';
```

Comment:
prosim dohodnete se s @krystof-k a @CoffeeladyCZ jestli budete pouzivat globalni enumy nebo ne. Me je to fuk co se pouzije.